source/coding-the-legislation/legislation_parameters.md

```diff
@@ -154,6 +154,84 @@ The scales built-in OpenFisca are:
 
 Example: [the french tax scale on income](https://fr.openfisca.org/legislation/impot_revenu.bareme)
 
+#### Ending a parameter at a specific date
+
+It is possible to end a parameter at a specific date simply by entering **null** as a value for that date.
+
+In the following example, the `housing allowance` is only defined from the 1st of January 2010 to the 30th of November 2016:
+
+File `parameters/benefits/housing_allowance.yaml`
```

Comment:
I see your point and I agree. I actually only added it to follow the same formula as the rest of the sections which gave YAML examples and indicated the file name.
Do you think we should git rid of the file and sample names for all of them @sandcha ?

Reply:
You're right, the [Parameter page](https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html) moves from `Sample file` to `Sample` to `File` 😆
I understand now that they are here to give titles to the displayed files contents. For now, I would update them all to `Sample file` and apply the same thing to the content introduced by this PR. 🤔

Reply:
Yes it is indeed quite confusing. I will update the titles on the [Parameter page](https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html), but I will not be adding a title to the the PR content since I moved it to the [Legislation evolution](https://openfisca.org/doc/coding-the-legislation/40_legislation_evolutions.html) page and the YAML files there are untitled for some reason 🤷‍♀️ 
Is that okay with you ?